Extensions/retags.js

```diff
@@ -102,13 +106,12 @@ XKit.extensions.retags = {
 				   $target = $element;
 				   url = $target.find('.action').data('post-url');
 			// Activity
-			} else if ($element.hasClass('ui_note')) {
-				if ($element.find('.part_response').length) {
-					$element.addClass('is_response');
-				}
+            } else if ($element.hasClass('activity-notification')) {
```

Comment:
Fallback is handled at line 46 where all logic was originally flowing. This PR just adds a new branch.